vendor/bat-native-ledger/src/bat/ledger/internal/publisher/server_publisher_fetcher.cc

```diff
@@ -55,45 +53,38 @@ void ServerPublisherFetcher::Fetch(
   FetchCallbackVector& callbacks = callback_map_[publisher_key];
   callbacks.push_back(callback);
   if (callbacks.size() > 1) {
-    BLOG(1, "Fetch already in progress for publisher " << publisher_key);
+    BLOG(1, "Fetch already in progress");
     return;
   }
 
-  BLOG(1, "Fetching server publisher info for " << publisher_key);
```

Comment:
this was removed so that we don't log browser history

Reply:
Could we print it for debug-only builds? Assuming we think it's useful, that is.

Reply:
we don't have debug build concept in shared lib :(